bedrock-devnet/devnet/__init__.py

```diff
@@ -297,7 +297,7 @@ def devnet_deploy(paths):
         log.info('Bringing up `op-challenger`.')
         run_command(['docker', 'compose', 'up', '-d', 'op-challenger'], cwd=paths.ops_bedrock_dir, env=docker_env)
 
-    # Optionally bring up Plasma Mode components.
+    # Optionally bring up Alt-DA Mode components.
     if DEVNET_PLASMA:
```

Comment:
I guess env vars like this will be changed in a follow-up?

Reply:
Yeah, we should do a separate pass later on to catch semi-visible things like this. The goal with this PR was to amend the CLI/Env flags for the binaries, as those are the most visible.

For `DEVNET_PLASMA`, there is also a `GENERIC_PLASMA` flag being used for different flag setting. It's something I'm not familiar enough with to make significant changes on without some more time reading and understanding.